src/calendar-list/index.js

```diff
@@ -294,7 +291,7 @@ class CalendarList extends Component {
 
   render() {
     return (
-      <View>
+      <View style={{flex: 1}}>
```

Comment:
What is this for?

Reply:
This is what actually fixes the rendering - `FlatList` scrolling only works on web if the parent component has a flex style. Source: https://github.com/necolas/react-native-web/issues/1436#issuecomment-612845122

Reply:
If that's a style to fix a web rendering you should add it to a StyleSheet with Platform division. You can't change for all Platforms to fix one...

Reply:
@tedsta Per @Inbal-Tish's comment, this should look something like:

```jsx
import { StyleSheet, Platform } from 'react-native'

...
render() {
  return <View style={Platform.select({ web: styles.webContainer, default: undefined })}>
}

const styles = StyleSheet.create({
  webContainer: {
    flex: 1
  }
})
```